common/src/main/java/com/mx/common/accessors/InvalidDataException.java

```diff
@@ -0,0 +1,18 @@
+package com.mx.common.accessors;
+
+import com.mx.common.exception.PathRequestException;
+
+/**
+ * Thrown when a request is correctly formed, but the data is invalid for some reason
+ *
+ * <p>See {@link PathRequestException} for usage details
+ */
+public class InvalidDataException extends AccessorUserException {
```

Comment:
Do you have an example use for this exception? It seems that an enhanced BadRequestException may be better. What is meant by "the date" a payload could have more than one date.

Reply:
An example from Qolo is the error "Request must provide only one of ('from_account_id', 'from_ach_account_id') and only one of ('to_account_id', 'to_ach_account_id')".

Another example would be "An amount $20 or greater is required."

Also, not sure what you mean by "the date". Did you mis-read "the data"?

Reply:
Yes, I misread it. haha.

Reply:
Since it extends AccessorUserException, which has report set to false, do you want me to also do it here?

Reply:
Just thinking about the name. Have you thought of any alternatives? Some that come to mind:

* InvalidRequestData
* RequestValidationException

What do you think, @gerfboy?

Reply:
I like RequestValidationException actually, I'll change to that.